WORKSPACE

```diff
@@ -3,38 +3,23 @@ workspace(
     name = "aspect_rules_js",
 )
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-
-# Override rules_nodejs to v6 to test the latest and recommended versional internally,
-# while keeping v5 in rules_js_dependencies() to avoid breaking changes.
-# TODO(2.0): change minimum to v6 in repositories.bzl
-http_archive(
-    name = "rules_nodejs",
-    sha256 = "a50986c7d2f2dc43a5b9b81a6245fd89bdc4866f1d5e316d9cef2782dd859292",
-    strip_prefix = "rules_nodejs-6.0.5",
-    url = "https://github.com/bazelbuild/rules_nodejs/releases/download/v6.0.5/rules_nodejs-v6.0.5.tar.gz",
-)
-
-load("//js:dev_repositories.bzl", "rules_js_dev_dependencies")
+load("@aspect_rules_js//js:dev_repositories.bzl", "rules_js_dev_dependencies")
 
 rules_js_dev_dependencies()
 
-load("//js:repositories.bzl", "rules_js_dependencies")
+load("@aspect_rules_js//js:repositories.bzl", "rules_js_dependencies")
 
 rules_js_dependencies()
 
-load("@aspect_bazel_lib//lib:repositories.bzl", "aspect_bazel_lib_dependencies", "register_jq_toolchains")
+load("@aspect_rules_js//js:configure.bzl", "rules_js_configure")
 
-aspect_bazel_lib_dependencies()
+rules_js_configure(node_version = "16.14.2")
```

Comment:
it bothers me that "configure" is invented here, isn't there an existing convention for that? let's look in some other languages

Reply:
There is `rules_go_register_toolchains` but that doesn't quite describe what I'm after here since it can be more than just toolchains that are configured.

Reply:
What else would we add other then registering toolchains?

Reply:
Just to outline and clarify what we have...

Today:
* `rules_js_dev_dependencies` which is internal repository deps, not public API so who cares
* `rules_js_dependencies` which is essentially repository deps
* `nodejs_register_toolchains` from rules_nodejs for node toolchains
* `npm_translate_lock` and `npm_import` have various `register_(copy_directory|copy_to_directory|coreutils|tar|yq)_toolchains` params to register bazel-lib toolchains
* ^ get set to false in many places, such as bzlmod wrappers of those, `pnpm_repository` etc

Future (bzlmod):
* All this goes away. Is anything remaining?

This PR:
* `rules_js_configure` with bool params and a lot of `node*` params to do all this? The `register_*` params go away.

Is that right?

Reply:
I'd like to suggest:
* Add `rules_js_register_toolchains(... all the bool params ...)` and all the `register_*` bools can be dropped
* Keep `nodejs_register_toolchains` as-is and don't put it in this new method, we can always think about it after an initial PR?

Reply:
Going with `rules_js_register_toolchains`